Classes/Command/ErrorPageCommandController.php

```diff
@@ -20,7 +20,10 @@
 use Neos\Utility\Files;
 use Netlogix\ErrorHandler\Configuration\ErrorHandlerConfiguration;
 use Netlogix\ErrorHandler\Service\ControllerContextFactory;
+use Netlogix\ErrorHandler\Service\DestinationResolver;
 use RuntimeException;
+use Symfony\Component\Yaml\Yaml;
```

Comment:
We should add `symfony/yaml` the composer.json

Reply:
Done.